specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip:checkNumberRouting": {
+      "get": {
+        "tags": [
+          "NumberRoutingValidation"
+        ],
+        "summary": "Gets the list of routes matching the target phone number, ordered by priority.",
+        "operationId": "CheckNumberRouting",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "targetPhoneNumber",
+            "description": "Phone number to test routing patterns against",
+            "required": true,
+            "type": "string",
+            "x-example": "+11234567890"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "query",
+            "name": "select",
```

Comment:
Is this feature valuable enough? TrunkRoute seems to have only a few properties and no heavy payloads.

Reply:
The `select` parameter will enable data to be much more lean - for the UI only the name is necessary.
Let's discuss on today's review

Reply:
I wonder what type we should use here. I've run a quick code search and it seems there are occurrences of both - [arrays](https://cs.github.com/Azure/azure-rest-api-specs/blob/a436672b07fb1fe276c203b086b3f0e0d0c4aa24/specification/search/data-plane/Azure.Search/stable/2019-05-06/searchindex.json?q=%22type%22%3A+%22array%22+select#L252) and [comma-separated strings](https://cs.github.com/Azure/azure-rest-api-specs/blob/a436672b07fb1fe276c203b086b3f0e0d0c4aa24/specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2018-05-05/AlertsManagement.json#L857).

However, according to the [guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options) the type should be a string array.

```json
"name": "$select",
"in": "query",
"type": "array",
"items": {
  "type": "string"
},
```

Reply:
@petrsvihlik I changed the type to array. Since it's formatted as a csv inside the query, it comes down to a comma-separated string, in practice

Reply:
Select removed as discussed during internal review meeting